pychunkedgraph/ingest/ran_agglomeration.py

```diff
@@ -24,6 +26,9 @@
 from ..graph.edges import EDGE_TYPES
 from ..graph.chunks.utils import get_chunk_id
 
+CRC_LENGTH = 4
```

Comment:
Please add a brief description of these parameters here.
Is it really okay to hardcode these values?

Reply:
I don't think @ranlu intends to change these often so it should be fine. Using all caps variable names to indicate constants.

If they do change often makes sense to use env vars or something similar. What is your opinion @ranlu ?

Reply:
I do not think I will move away from crc32 any time soon, I switch to crc32c to use faster instructions and more robust error detection, but that would not change the length of the checksum.